zjit/src/hir.rs

```diff
@@ -2804,7 +2803,7 @@ mod tests {
         assert_method_hir_with_opcode("test", YARVINSN_newarray, expect![[r#"
             fn test:
             bb0(v0:BasicObject, v1:BasicObject, v2:BasicObject):
-              v5:ArrayExact = NewArray v1, v2
+              v5:ArrayExact = NewArray v0, v1
```

Comment:
This is `[self, a]` instead of `[a, b]`

Reply:
Thanks, your patch fixed it https://github.com/ruby/ruby/pull/13529/commits/5e62c53d9685bdddd4239a31cbe7a4d67fa1f9db